AppleALC/kern_alc.cpp

```diff
@@ -59,7 +59,7 @@ void AlcEnabler::updateProperties() {
 
 		// Respect desire to disable digital audio. This may be particularly useful for configurations
 		// with broken digital audio, resulting in kernel panics. Ref: https://github.com/acidanthera/bugtracker/issues/513
-		if (hasBuiltinDigitalAudio && devInfo->audioBuiltinAnalog && devInfo->audioBuiltinAnalog->getProperty("No-hda-gfx"))
+		if (hasBuiltinDigitalAudio && devInfo->audioBuiltinAnalog && checkKernelArgument("-alc-disable-digital-audio"))
```

Comment:
This **replaces** property implemented by Apple, which we are obliged to properly interpret, with some random boot argument. If anything, both should be supported. And using such a long name makes no sense to me. Use something like `-alcnohdau` if you really want it.

Reply:
Ok. I will use -alcnohdau here, and keep the "No-hda-gfx" check.

Reply:
Done